build/packaging/linux/bin/ksend

```diff
@@ -0,0 +1,254 @@
+}
+
+__kill_timeout() {
+    local PIDNUM=$1
+    kill $PIDNUM
+    for i in `seq 0 100 $((1000 * $STOP_TIMEOUT))`; do
+        if ! kill -0 $PIDNUM 2> /dev/null; then
+            echo "OK"
+            return
+        fi
+        sleep 0.1
+    done
+    kill -9 $PIDNUM && echo "Killed"
+}
+
+start() {
+    __pid_run
+    [ -n "$pid" ] && echo "ksend already running...[$pid]" && return
+
+    echo -n "Starting ksend: "
+
+    if [ ! -d $LOG_DIR ]; then
+        mkdir -p $LOG_DIR
+    fi
+
+    set -f
+    OPTIONS=""
+
+    if [[ ! -z $METRICS ]] && [[ $METRICS -eq 1 ]]; then
+        OPTIONS="$OPTIONS --metrics"
+    fi
+
+    if [[ ! -z $PROMETHEUS ]] && [[ $PROMETHEUS -eq 1 ]]; then
+        OPTIONS="$OPTIONS --prometheus"
+    fi
+
+    if [[ ! -z $DB_NO_PARALLEL_WRITE ]] && [[ $DB_NO_PARALLEL_WRITE -eq 1 ]]; then
+        OPTIONS="$OPTIONS --db.no-parallel-write"
+    fi
+
+    if [[ ! -z $MULTICHANNEL ]] && [[ $MULTICHANNEL -eq 1 ]]; then
+        OPTIONS="$OPTIONS --multichannel"
+    fi
+
+    if [[ ! -z $RPC_ENABLE ]] && [[ $RPC_ENABLE -eq 1 ]]; then
+        OPTIONS="$OPTIONS --rpc"
+        if [ ! -z $RPC_API ]; then
+            OPTIONS="$OPTIONS --rpcapi $RPC_API"
+        fi
+        if [ ! -z $RPC_PORT ]; then
+            OPTIONS="$OPTIONS --rpcport $RPC_PORT"
+        fi
+        if [ ! -z $RPC_ADDR ]; then
+            OPTIONS="$OPTIONS --rpcaddr $RPC_ADDR"
+        fi
+        if [ ! -z $RPC_CORSDOMAIN ]; then
+            OPTIONS="$OPTIONS --rpccorsdomain $RPC_CORSDOMAIN"
+        fi
+        if [ ! -z $RPC_VHOSTS ]; then
+            OPTIONS="$OPTIONS --rpcvhosts $RPC_VHOSTS"
+        fi
+    fi
+
+    if [[ ! -z $WS_ENABLE ]] && [[ $WS_ENABLE -eq 1 ]]; then
+        OPTIONS="$OPTIONS --ws"
+        if [ ! -z $WS_API ]; then
+            OPTIONS="$OPTIONS --wsapi $WS_API"
+        fi
+        if [ ! -z $WS_PORT ]; then
+            OPTIONS="$OPTIONS --wsport $WS_PORT"
+        fi
+        if [ ! -z $WS_ADDR ]; then
+            OPTIONS="$OPTIONS --wsaddr $WS_ADDR"
+        fi
+        if [ ! -z $WS_ORIGINS ]; then
+            OPTIONS="$OPTIONS --wsorigins $WS_ORIGINS"
+        fi
+    fi
+
+    # Cypress network => NETWORK_ID is null && NETWORK = "cypress"
+    # Baobab network => NETWORK_ID is null && NETWORK = "baobab"
+    # Else => private network
+    if [[ -z $NETWORK_ID ]]; then
+        if [[ $NETWORK == "baobab" ]]; then
+            OPTIONS="$OPTIONS --baobab"
+        elif [[ $NETWORK == "cypress" ]]; then
+            OPTIONS="$OPTIONS --cypress"
+        else
+            echo
+            echo "[ERROR] network id is not specified and network is not available."
+            echo "Available network: baobab, cypress"
+            exit 1
+        fi
+    else
+        OPTIONS="$OPTIONS --networkid $NETWORK_ID"
+        echo "[INFO] creating a private network: $NETWORK_ID"
+        if [[ ! -z $NETWORK ]]; then
+            echo
+            echo "[WARN] ignoring the specified network: $NETWORK"
+        fi
+    fi
```

Comment:
In service chain, do we need to prevent baobab/cypress options?

Reply:
I suggest like below. Please consider it.
- Remove NETWORK option.
- If there is no --networkID value, it can exit and echo warn message.